### ISSUE TITLE
Ubuntu 23.10 snap opens the first time, but never a second

### ISSUE THREAD

**Issue by thekswenson** [bitwarden/clients#7890]
### Steps To Reproduce

1. Install the bitwarden snap
2. Open bitwarden
3. Close bitwarden
4. Try to open bitwarden again (through terminal or icon)

### Expected Result

Expected to see bitwarden open

### Actual Result

Silently does nothing.
Any call to bitwarden on the commandline return nothing (e.g. `bitwarden --help` give nothing).

Uninstalling an reinstalling make bitwarden work gain, but only once.

### Additional Context

This may have something to do with https://github.com/bitwarden/clients/issues/2629. While the workaround of calling `/snap/bitwarden/current/bitwarden` directly does work, the other workaround of `rm ~/snap/bitwarden/current/.config/user-dirs.*` does not.

It was difficult to get the build number because the "about" window gives only boxes where each character should be. I found it in the Ubuntu  software center.

![Screenshot from 2024-02-09 09-38-58](https://github.com/bitwarden/clients/assets/9959942/11945e49-813f-4f35-873c-3730151027aa)

### Operating System

Linux

### Operating System Version

Ubuntu 23.10

### Installation method

Snap

### Build Version

2024.2.0

### Issue Tracking Info

- [X] I understand that work is tracked outside of GitHub. A PR will be linked to this issue should one be opened to address it, but Bitwarden doesn't use fields like "assigned", "milestone", or "project" to track progress.

**Comment by NovaSilentium** [comment]
Hi there,

I am unable to reproduce this issue, it has been escalated for further investigation. If you have more information that can help us, please add it below. 

Thanks!

**Comment by okarmusk** [comment]
I have similar problem, but not sure if it is the same, anyway I opened issue where I have added logs from terminal:
https://github.com/bitwarden/clients/issues/8175
For development team logs should be more verbose.

**Comment by closebot-bw** [comment]
⚠️ **Stale Issue Notice**

This issue has been automatically marked as stale due to inactivity. It will be closed in **2 weeks** (October 15, 2025) if no further activity occurs.

If this issue is still relevant and you would like to keep it open, please:
- Comment on this issue to show continued interest
- Provide any additional information or updates
- Confirm that the issue still exists in the latest version

Thank you for your contribution to this project! 🙏

**Comment by closebot-bw** [comment]
🔒 **Issue Closed Due to Inactivity**

This issue has been automatically closed due to lack of activity for an extended period. We periodically review and close inactive issues to help maintain our issue tracker and focus on current priorities.

**If this issue is still relevant:**
- Please create a new issue with updated information
- Include steps to reproduce the problem if it's a bug report
- Mention if this issue still occurs in the latest version

Thank you for your contribution to this project. Your feedback helps us improve! 🙏
